checkarg/__init__.py

```diff
@@ -0,0 +1,2 @@
+__doc__ = "__init__"
+__version__ = "0.0.1"
```

Comment:
Shouldn't this version propagate throughout the setup files. Maybe even along with other metadata

Reply:
It's automatically generated by bump2version. this tool controls the following files in terms of versioning:
* setup.py
* setup.cfg
* checkarg/__initi__.py